Our compiler will think that os_type = "Win32".  Unfortunately in
the default OCaml this has the negative effect of causing it to
use '\' character in paths.  Since it's really running on a Linux
kernel, that won't work.  This is a quick and dirty fix.

Index: mingw-ocaml/build/ocaml/stdlib/filename.ml
===================================================================
--- mingw-ocaml.orig/build/ocaml/stdlib/filename.ml	2013-03-22 13:57:37.792641647 -0500
+++ mingw-ocaml/build/ocaml/stdlib/filename.ml	2013-03-22 13:57:37.776649648 -0500
@@ -93,7 +93,7 @@
 module Win32 = struct
   let current_dir_name = "."
   let parent_dir_name = ".."
-  let dir_sep = "\\"
+  let dir_sep = "/"
   let is_dir_sep s i = let c = s.[i] in c = '/' || c = '\\' || c = ':'
   let is_relative n =
     (String.length n < 1 || n.[0] <> '/')
